_opensearch/configuration.md

```diff
@@ -65,6 +65,7 @@ PUT _cluster/settings
 }
 ```
 
+For further information see the [Cluster settings API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/) documentation.
```

Comment:
```suggestion

## Next steps

For more cluster settings, see the [Cluster settings API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/) documentation.
```

Reply:
i disagree with the "for more cluster settings" wording: the documentation to which i link doesn't describe the cluster settings but instead describes the REST API in more details.

Reply:
thanks, i've applied that. was the bullet point intentional? it looked a bit wrong, so i left it away for the moment.
otherwise: you (as a maintainer) have write access to the branch of this PR, so feel free to add it in directly so you don't have to wait for me to do it